index.css

```diff
@@ -358,7 +453,269 @@ main {
   display: none;
 }
 
+/* News section */
+
+#news-cont-last {
+  display: none;
+}
+
+.news-main {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.news-cont {
+  height: 450px;
+  width: 100%;
+}
+
+.news-cont h2 {
+  height: 5%;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.news-cont-title {
+  color: var(--orange);
+  display: inline;
+  float: left;
+}
+
+.news-cont-img {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/gamestop.jpg);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 12px;
+}
+
+.news-cont-img2 {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/news-img-2.png);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 12px;
+}
+
+.news-cont-img3 {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/news-img-3.jpg);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 12px;
+}
+
+.news-cont-img4 {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/news-img-4.jpg);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 12px;
+}
+
+.news-cont-img5 {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/news-img-2.png);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 12px;
+}
+
+.news-cont-img6 {
+  height: 35%;
+  width: 100%;
+  background: linear-gradient(rgba(255, 255, 255, 0.2), rgba(0, 0, 0, 1)), url(img/news-img-6.jpg);
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
```

Comment:
[Optional]
You have repeated almost all the properties in `.news-cont-imgx` classes where x = 2,3,4,5,6,7,8,9. you can use DRY principle like this.

```
.news-cont-img,
.news-cont-img2,
.news-cont-img3
.news-cont-img4,
......,
 {
  height: 35%;
  width: 100%;
  background-position: center;
  background-size: cover;
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  padding: 12px;
}
```